js/utility/formatters.js

```diff
@@ -0,0 +1,58 @@
+namespacer('baltimoreCounty.utility');
+
+baltimoreCounty.utility.format = (function () {
+    var _formatters = {
+        currency: function formatCurrency(input) {
```

Comment:
Not a big deal, but would you move these functions out of your object declaration? For example:
```
function whatnot() {}

var thing = { 
    stuff: whatnot 
};
```